components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -162,8 +162,9 @@ func main() {
 		// TODO(workaround until #1049): following components should be always disabled and user should not be able to enable them in provisioning request. This implies following components cannot be specified under the plan schema definition.
 		"BackupInt":               runtime.NewGenericComponentDisabler("backup-init", "kyma-system"),
 		"Backup":                  runtime.NewGenericComponentDisabler("backup", "kyma-system"),
-		"KnativeProvisionerNatss": runtime.NewGenericComponentDisabler("knative-provisioner-natss", "knative-eventing"),
-		"NatssStreaming":          runtime.NewGenericComponentDisabler("nats-streaming", "natss"),
+		"KnativeEventingKafka":    runtime.NewGenericComponentDisabler(provisioning.KymaComponentNameKnativeEventingKafka, "knative-eventing"),
```

Comment:
Can we make this a const and use it everywhere? :) Same for the Natss entries. Doing so prevents us from having typos when refering to the component